File: languages/java/Item.java
public class Item {
  private String name;
  private int shelfLife;
  private int worth;

  public Item(String name, int shelfLife, int worth) {
    this.name = name;
    this.shelfLife = shelfLife;
    this.worth = worth;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public int getShelfLife() {
    return shelfLife;
  }

  public void setShelfLife(int shelfLife) {
    this.shelfLife = shelfLife;
  }

  public int getWorth() {
    return worth;
  }

  public void setWorth(int worth) {
    this.worth = worth;
  }
}

File: languages/python/item.py
class Item:
    def __init__(self, name, shelf_life, worth):
        self.name = name
        self.shelf_life = shelf_life
        self.worth = worth

    def get_name(self):
        return self.name

    def set_name(self, name):
        self.name = name

    def get_shelf_life(self):
        return self.shelf_life

    def set_shelf_life(self, shelf_life):
        self.shelf_life = shelf_life

    def get_worth(self):
        return self.worth

    def set_worth(self, worth):
        self.worth = worth
